### PR TITLE
Plan/pricing updates

### PR DESCRIPTION
**Pull request by dbanys**

### CODE REVIEW

File: src/docs/reference/pricing.md [hunk 1]
```diff
@@ -2,37 +2,225 @@
...
+
+In addition to usage-based pricing, you will need a basic subscription to use Railway. These plans are detailed below.
+
+## Free Trial
+
+As a new user, you can take the platform for a spin by starting a free trial. The trial gives access to basic features and includes a one-time grant of $5 for usage.
+
+Full Trial users can deploy code and plugins, while Limited Trial users can only deploy plugins.
+
+Your trial experience depends on whether Railway can verify your account. Read more about verification (link to below).
```
Comment (half0wl) [inline src/docs/reference/pricing.md:17]: This should link to the verification section below

File: src/docs/reference/pricing.md [hunk 2]
```diff
@@ -2,37 +2,225 @@
...
+- Pro — For professional developers and their teams shipping to production
+- Enterprise — For dev teams building and deploying production applications with the need for enterprise features related to compliance, SLAs, and account management
+
+## Plan Pricing
+
+- Hobby → $5 / mo, including $5 of usage
+- Pro → $20 / seat / mo
+- Enterprise → Custom pricing
+
+Read more about the plans here: railway.app/pricing.
```
Comment (half0wl) [inline src/docs/reference/pricing.md:33]: Should be a direct link to the page

File: src/docs/reference/pricing.md [hunk 3]
```diff
@@ -2,37 +2,225 @@
...
-  <br /><br />For full details, read the [blogpost](https://blog.railway.app/p/introducing-trial-hobby-pro-plans) and check out the [Migration Guide](https://blog.railway.app/p/pricing-and-plans-migration-guide-2023). 
+Keep in mind, if your project has PR deploys enabled, Railway deploys a mirror copy of your service(s) based on the environment it forks from (`production` by default). You are billed for the services that are spun while the ephemeral environment is active.
 </Banner>
 
-Railway offers a consumption-based pricing model for your projects. You don't get charged for resources you don't use, instead, Railway charges by the minute for each vCPU and memory resource your service uses.
+## Add-on services
 
-[You can see the offerings of each tier on this page.](/reference/plans)
+Railway offers [Business Class Support](https://www.notion.so/5c47cc42b2cb456c85d125b512bb8763?pvs=21) as an add-on service for teams on the Pro plan (Business Class Support is included with Enterprise). [Contact us](mailto:contact@railway.app) to get started.
```
Comment (half0wl) [inline src/docs/reference/pricing.md:55]: Replace notion link w/  `/reference/support#business-class`

File: src/docs/reference/pricing.md [hunk 4]
```diff
@@ -2,37 +2,225 @@
...
+- If you cancel your Hobby plan subscription, any remaining credit balance will be lost
 
 ## Billing Management
 
-You can add and manage your payment info under the billing page and see historical billing usage under the account billing page: [railway.app/account/billing](https://railway.app/account/billing)
+You can add and manage your payment info under the billing page and see historical billing usage under the account billing page: [railway.app/account/billing](https://railway.app/account/billing)
+
+Within this page, you can also top-up on credits for your account if you prefer to pay for the Hobby plan in a prepaid fashion.
+
+For more information, read the [Billing](https://docs.railway.app/reference/pricing#billing-management) docs.
```
Comment (half0wl) [inline src/docs/reference/pricing.md:78]: This links back to this section. I think we should remove it?

File: src/docs/reference/pricing.md [hunk 5]
```diff
@@ -2,37 +2,225 @@
...
+
+## Verification
+
+Verification is a fraud prevention measure designed to filter out spammy actors while incentivizing legitimate actors.
+
+Verification status depends on a number of factors, including the age and activity of your GitHub account.
+
+If you have trouble getting verified (often because your GitHub account is not up to the verification standard), don’t fret! You are still able to use a credit card and purchase Railway plans.
```
Comment (half0wl) [inline src/docs/reference/pricing.md:86]: "... and purchase Railway plans" -- "Railway plans" should ideally link to `/reference/pricing#plans`

File: src/docs/reference/pricing.md [hunk 6]
```diff
@@ -2,37 +2,225 @@
...
+
+### Usage
+
+**How is usage billed?**
+
+    Railway charges for the core components of compute that you consume. These include:
+
+    - GB RAM used. Billed by the minute.
+    - Cores of vCPU used. Billed by the minute.
+    - GB of Network Egress. Billed by the kb.
```
Comment (half0wl) [inline src/docs/reference/pricing.md:151]:
```suggestion
    - RAM used. Measured in GB, billed by the minute.
    - vCPU used. Measured in cores, billed by the minute.
    - Network Egress bandwidth used. Measured in GB, billed per-kb.
```

File: src/docs/reference/pricing.md [hunk 7]
```diff
@@ -2,37 +2,225 @@
...
+
+    Railway offers additional add-ons that are usage-based:
+
+    - GB Volume. Billed by the minute.
```
Comment (half0wl) [inline src/docs/reference/pricing.md:155]:
```suggestion
    - Volumes. Measured in GB, billed by the minute.
```

File: src/docs/reference/pricing.md [hunk 8]
```diff
@@ -2,37 +2,225 @@
...
+
+    Prices are listed here.
```
Comment (half0wl) [inline src/docs/reference/pricing.md:157]: should link to https://railway.app/pricing

File: src/docs/reference/pricing.md [hunk 9]
```diff
@@ -2,37 +2,225 @@
...
+
+**What will it cost to run my app?**
+
+    With Railway, you are only billed for the resources you consume at any given time. If you want to approximate the cost of running your app, try our calculator.
```
Comment (half0wl) [inline src/docs/reference/pricing.md:161]: "try our calculator" should link to the calculator on pricing page

File: src/docs/reference/pricing.md [hunk 10]
```diff
@@ -2,37 +2,225 @@
...
+    Usage limits are coming to Railway. In the meantime, if you are on the Hobby plan and you want a guarantee that you keep under a certain spending threshold, you can set up credits as your payment method.
+
+### Billing
+
+**What is the difference between subscription and usage?**
+
+    There are two main components to your bill:
+
+    - **Subscription**: Cost of the plan you’re on (`cost per seat` * `purchased seats`)
+    - **Usage**: Cost of the resources you’ve consumed: (`cost per unit` * `used units`)
```
Comment (half0wl) [inline src/docs/reference/pricing.md:182]:
```suggestion
    - **Usage**: Cost of the resources you’ve consumed (`cost per unit` * `used units`)
```

File: src/docs/reference/pricing.md [hunk 11]
```diff
@@ -2,37 +2,225 @@
...
+
+**How do I view and manage my subscription?**
+
+    To view and manage your subscription, visit your account page.
```
Comment (half0wl) [inline src/docs/reference/pricing.md:186]: Should link to /account/billing

Comment (half0wl) [reply src/docs/reference/pricing.md:186]: Perhaps rephrase to "Visit the billing section of the Railway dashboard" for consistency?

File: src/docs/reference/pricing.md [hunk 12]
```diff
@@ -2,37 +2,225 @@
...
+
+    When you make a payment, you will receive an email with your invoice and receipt attached. You can also find your billing history in the [billing section](https://railway.app/account/billing) of the Railway dashboard.
+
+**How can I add company information to my invoice?**
+
+    Head to the [billing section](https://railway.app/account/billing) in your dashboard and click “Manage Subscription.” From there, you can update your billing information, including billing address and Tax ID.
+
+**What payment methods are accepted?**
+
+    Railway accepts credit cards for plan subscriptions, usage, and to purchase prepaid credits. We support invoice payments for customers on the Enterprise plan.
```
Comment (half0wl) [inline src/docs/reference/pricing.md:198]: > invoice payments

should clarify difference between this and credit cards (i.e. ability to pay over direct wire transfer etc.)

File: src/docs/reference/pricing.md [hunk 13]
```diff
@@ -2,37 +2,225 @@
...
+
+**Can I try Railway without a credit-card?**
+
+    Yes. As a new Railway user you can sign up for a free Trial. You will receive a one-time grant of $5 to use on resources.
+
+**I am a freelancer or represent an agency. How do I manage my billing relationships with my clients?**
+
+    Create a Pro plan on Railway and add the client to the team. If you run into issues when it’s time to pass on your workload to your client, email us — and we can help onboard your client to Railway.
```
Comment (half0wl) [inline src/docs/reference/pricing.md:206]: "email us" should link out to our contact email

File: src/docs/reference/pricing.md [hunk 14]
```diff
@@ -2,37 +2,225 @@
...
+
+**I prefer to prepay. Is that possible?**
+
+    Yes. You can use prepaid credits as a payment method on Railway if you prefer to prepay for Railway’s services. You will still need to pay a monthly subscription fee, as well as for any usage. Those amounts will be deducted from your credit balance.
+
+**What happens if I use credits as a payment method and my account runs out of credits?**
+
+    If you are using credits as a payment method and you credit balance reaches zero, you will no longer be able to deploy to Railway and we will stop all of your workloads. You will have a grace period to add new credits before we purge your data.
```
Comment (half0wl) [inline src/docs/reference/pricing.md:214]:
```suggestion
    If you are using credits as a payment method and your credit balance reaches zero, you will no longer be able to deploy to Railway, and we will stop all of your workloads. You will have a grace period to add new credits before we purge your data.
```

File: src/docs/reference/pricing.md [hunk 15]
```diff
@@ -2,37 +2,225 @@
...
+
+### General
+
+**Can I upgrade or downgrade at any time?**
 
-Within this page, you can also top up on credits for your account if you prefer to pay for the Hobby Plan in a prepaid fashion.
+    You can upgrade any time, and when you do, you will get to the features of your new plan, as well as access to more powerful resources, immediately. When you downgrade, the changes will take effect at the beginning of your next billing cycle.
```
Comment (half0wl) [inline src/docs/reference/pricing.md:220]:
```suggestion
    You can upgrade any time, and when you do, you will get access to the features of your new plan, as well as access to more powerful resources immediately. When you downgrade, the changes will take effect at the beginning of your next billing cycle.
```

**Review by half0wl — CHANGES_REQUESTED** [review]

File: src/docs/reference/teams.md [hunk 16]
```diff
@@ -2,16 +2,14 @@
 title: Teams
 ---
 
-Teams are how organizations are represented within Railway. Teams with a valid credit card attached are subscribed to the Team plan. Projects within the Team plan get generous limits on service caps such as 32 GB of RAM and 32 vCPU cores.
+Teams are how organizations are represented within Railway. Teams can be created via the Hobby, Pro, or Enterprise plans.
```
Comment (ohrgaard) [inline src/docs/reference/teams.md:5]: >Teams can be created via the Hobby, Pro, or Enterprise plans.

Teams can only be created for Pro and Enterprise, right?

Comment (half0wl) [reply src/docs/reference/teams.md]: fixed in 9f1b7b0f0b4811e245fedee26b205b8df461f515

**Pull request merged**